web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/add-ons.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 'gatsby-wordpress-add-ons'
+title: 'Add Ons'
+slug: '/frontend-starters/nextjs/gatsby/gatsby-wordpress/add-ons'
+sidebar_position: 5
+---
+
+### Advanced Custom Fields
+
+Advanced Custom Fields turns WordPress sites into a fully-fledged content
+management system by giving you all the tools to do more with your data.
+
+Advanced custom fields let you add, store, and display additional information
+about a piece of content in WordPress. This information can easily be assessed
+through GraphQL with simple adjustments to existing queries.
+
+This add-on uses GraphQL to bring in a custom `related content` data type that
+was created with Advanced Custom Fields. Your starter kit will have a related
```

Comment:
```suggestion
was created with Advanced Custom Fields. After adding the `gatsby-wp-acf-addon`, our starter kit will have a related
```
More specificity here - otherwise it reads as though the starter kit has this already.